apps/dotcom/client/src/tla/utils/copy.ts

```diff
@@ -0,0 +1,11 @@
+export async function copyTextToClipboard(text: string) {
```

Comment:
feels like this would be just useful to expose (either by putting it in packages/utils or from tldraw, either way)

Reply:
(to avoid duplication)

Reply:
It kind of is a duplication, though the one in the tldraw library takes a blob rather than a string.